### ISSUE TITLE
Support to use custom table name instead of class name

### ISSUE THREAD

**Issue by gustavolealg** [HeadspringLabs/bulk-writer#14]
Please add support to use custom table neame instead of class name.

Something like:
`public BulkWriter(string connectionString, string tableName = "")`

and bellow:

`var destinationTableName = (tableName.Trim() != string.Empty ) ? tableName : typeof(TResult).GetTypeInfo().GetCustomAttribute<TableAttribute>()?.Name ?? typeof(TResult).Name;`

**Comment by caioproiete** [comment]
@gustavolealg We already have custom table name support with the `TableAttribute` as [you can see in the demo project](https://github.com/HeadspringLabs/bulk-writer/blob/master/src/BulkWriter.Demo/MyDomainEntity.cs#L6).

Are you looking for defining the table name dynamically/on-the-fly?

**Comment by gustavolealg** [comment]
@caioproiete yes, I was talking about that, but I did a test and Schema attribute is not working, allways looking for the table in dbo context. Can this be fixed?

Regards

**Comment by caioproiete** [comment]
@gustavolealg Having support for custom table schema other than `dbo` is resolved by pull-request #13. Just waiting for it to get reviewed & merged.

However, there's a valid use-case for dynamically defining the schema & table name on-the-fly (vs. defining it statically with the `TableAttribute`). Will open a new PR after #13 gets merged.
